integration_tests/integration_tests.feature

```diff
@@ -35,6 +35,18 @@ Feature: Run integration tests
      |        default_5       |    24    |      60     |      60     |
      |        default_csv     |    24    |      60     |      60     |
 
+  Scenario Outline: Test Balanced Energy Bills
+     Given we have a scenario named <scenario>
+     And d3a is installed
+     When we run the d3a simulation with <scenario> [24, 60, 60]
+     Then we test energy bills
```

Comment:
In order for the step to denote exactly what it does, I would change it to "Then the energy bills report the correct accumulated energy values".